common/src/main/java/com/revenuecat/purchases/common/PurchaseHistoryRecordWrapper.kt

```diff
@@ -2,35 +2,26 @@ package com.revenuecat.purchases.common
 
 import com.android.billingclient.api.PurchaseHistoryRecord
 import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.PurchaseDetails
+import com.revenuecat.purchases.models.PurchaseType
 import com.revenuecat.purchases.models.RevenueCatPurchaseState
 import org.json.JSONObject
 
-class PurchaseHistoryRecordWrapper(
-    override val type: ProductType,
-    override val purchaseToken: String,
-    override val purchaseTime: Long,
-    override val sku: String,
-    override val presentedOfferingIdentifier: String? = null,
-    override val purchaseState: RevenueCatPurchaseState,
-    override val storeUserID: String?,
-    override val isAutoRenewing: Boolean?,
-    override val signature: String?,
-    override val orderId: String = purchaseToken,
-    override val originalJson: JSONObject?
-) : PurchaseWrapper {
-
-    constructor(
-        purchaseHistoryRecord: PurchaseHistoryRecord,
-        type: ProductType
-    ) : this(
+fun PurchaseHistoryRecord.toRevenueCatPurchaseDetails(
```

Comment:
maybe this file should be named something like `PurchaseHistoryRecordToPurchaseDetailsConversion` (or maybe something simpler 😂 )

Reply:
I am going to put it together with the other Google Purchase conversion. I realized this doesn't have to live in the common module either, so I will move it to :feature:google. Win win